examples/tcp_echo_workload/main.rs

```diff
@@ -108,10 +109,12 @@ async fn main() -> anyhow::Result<()> {
 
     let mut trace_level = Tracing::Off;
     let mut bootstrap_address = "[::1]:1234".parse()?;
+    let mut enable_probes = false;
     for arg in &args[1..] {
         match arg.as_str() {
             "--tracing" => trace_level = Tracing::On,
             "--super-tracing" => trace_level = Tracing::ReallyOn,
+            "--probes" => enable_probes = true,
```

Comment:
why require the flag? I don't really have an opinion, just curious

Reply:
For consistency, and that, right now, registration doesn't happen automatically. We use this to gate the call to register them.